spec/occi/core/locations_spec.rb

```diff
@@ -0,0 +1,111 @@
+module Occi
+  module Core
+    describe Locations do
+      subject(:locs) { locations }
+
+      let(:locations) { Locations.new(uris: uris) }
```

Comment:
Why do you need this `let` statement? Why don't you initialize the subject in its own block? (same for the TR tests)